SECURITY.md

```diff
@@ -3,6 +3,23 @@
 
 Join the [kubestellar-security-announce](https://groups.google.com/u/1/g/kubestellar-security-announce) group for emails about security and major API announcements.
 
+## Dependencies Policy
+
+KubeStellar manages its dependencies with the following policy:
+
+- **Dependency Detection:** We use [Dependabot](https://github.com/dependabot) to automatically check for and propose updates to dependencies in Go modules, Python requirements, Dockerfiles, and Helm charts. Dependabot PRs serve as prompts but are not automatically accepted.
+- **Update Process:** After Dependabot creates a PR, maintainers wait for potential issues to surface before proceeding. They then create their own PR that follows our [GitHub Action reference discipline](https://github.com/kubestellar/kubestellar/blob/main/CONTRIBUTING.md#github-action-reference-discipline) and other established practices.
```

Comment:
This is accurate in the case of an update to the version of an Action in a GitHub Actions workflow. This is _not_ necessarily accurate in the case of an update to a Go-language dependency. In that scenario, the handling depends on whether the update proposed by Dependabot is functional or not. We have seen Dependabot make broken proposals, and we have seen it make good ones. In the case of a broken proposal, we will make our own PR that covers the change.